routes/routes.go

```diff
@@ -264,7 +264,7 @@ func SetAuthorizedRoutes(handler *gin.Engine, middlewares ...gin.HandlerFunc) {
 			segments.DELETE("/:id", actions.DeleteSegment)
 			segments.PUT("/:id/subscribers", actions.PutSegmentSubscribers)
 			segments.GET("/:id/subscribers", middleware.PaginateWithCursor(), actions.GetSegmentsubscribers)
-			segments.DELETE("/:id/subscribers", actions.DetachSegmentSubscribers)
+			segments.POST("/:id/subscribers", actions.DetachSegmentSubscribers)
```

Comment:
We should leave this method to be DELETE since we are removing all subscribers from this segment.

Reply:
https://stackoverflow.com/questions/299628/is-an-entity-body-allowed-for-an-http-delete-request 
We send ids to the request